### PR TITLE
Adds Caddy and generic-terms to README

### PR DESCRIPTION
**Pull request by ConnorVG**
Apache and Nginx aren't the only Http servers, damn it! :laughing:

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I think we didn't go with cady because these docs were targeted at production use rather than development.

**Comment by ConnorVG** [comment]
What about people who develop in production? :trollface:

**Comment by GrahamCampbell** [comment]
Haha.

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Cachet is a beautiful and powerful open source status page system.
 ## Requirements
 
 - PHP 5.5.9+ or newer
-- Apache or Nginx server
+- Http server with PHP support (eg: Apache, Nginx, Caddy)
```
Comment (jbrooksuk) [inline README.md]: HTTP*

;)

Comment (GrahamCampbell) [inline README.md]: h2 :trollface:

Comment (ConnorVG) [inline README.md]: Is it just me who hates reading `HTTP` vs `Http`? It's so much nicer as a more _wordy word_... yakno

Comment (jbrooksuk) [inline README.md]: Yes.

Comment (ConnorVG) [inline README.md]: You're not supposed to say yes, damn it.

Comment (GrahamCampbell) [inline README.md]: just write "A TCP server with HTTP support". :trollface:

Comment (ConnorVG) [inline README.md]: We could just cover all grounds and say `something that does some Http stuff with PHP, innit`

Comment (ConnorVG) [inline README.md]: image

**Comment by ConnorVG** [comment]
I WIN

**Comment by jbrooksuk** [comment]
I changed it.

**Comment by ConnorVG** [comment]
damn

**Pull request merged**